frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortletUtils.java

```diff
@@ -363,4 +368,50 @@ public static VerificationState getVerificationState(VulnerabilityDTO vul){
                                 .findFirst()
                                 .orElse("")));
     }
+
+    public static Map<String, Integer> getServeListMapFromRequest(ResourceRequest request) {
```

Comment:
why
<pre>
    public static Map<String, Integer> getServe<b>List</b>MapFromRequest(ResourceRequest request) {
</pre>
I can't see the "List"

Reply:
This is a rather generic function name, which does not help me with understanding the functionallity

Reply:
changed to `getDatatablePaginationAttributeMapFromRequest`